tests/v2/validation/provisioning/provisioning_node_driver_test.go

```diff
@@ -303,11 +140,11 @@ func (r *RKE2NodeDriverProvisioningTestSuite) TestProvisioning_RKE2AWSCluster()
 					testSessionClient, err := tt.client.WithSession(testSession)
 					require.NoError(r.T(), err)
 
-					clusterName := AppendRandomString(baseAWSClusterName)
+					clusterName := AppendRandomString("auto-" + provider.Name + "-")
```

Comment:
I think it would look cleaner if the `"auto-"+ provider.Name +"-"` all happened in `AppendRandomString`, I don't see a problem with that. What do you think?